docs/components/modeler/web-modeler/git-sync.md

```diff
@@ -0,0 +1,85 @@
+---
+id: git-sync
+title: Git sync
+description: Sync your Git repositories with Web Modeler.
+---
+
+Organization owners and administrators can connect their Web Modeler projects to GitHub, allowing users to keep their Web Modeler, Desktop Modeler, and official version control projects synced. Once the basic integration is configured by an organization owner or organization administrator, project administrators and editors can use the built-in button to pull changes from GitHub, integrate contributions from Desktop Modeler users, and merge their own work.
+
+## Connect to GitHub
+
+### Create a new GitHub App
+
+Web Modeler requires a GitHub App to sync changes with your GitHub repository.
+
+1. Follow the [GitHub documentation](https://docs.github.com/en/apps/creating-github-apps/registering-a-github-app/registering-a-github-app) to create a new GitHub App for your organization or account with the following configuration:
+
+   - Under **Webhooks**, deselect **Active**
+   - Under **Permissions > Repository permissions**, enable **Read and write** for the following options:
+     - Commit statuses
+     - Contents
+     - Pull requests
+
+   Click **Create GitHub App** to finish.
+
+2. In your new application's setting page, navigate to **General > Private keys**, and select **Generate a private key**. This key will be needed for Web Modeler to access your repository.
+
+### Configure GitHub in Web Modeler
+
+:::note
+
+An organization administration account (or project administrator in Camunda Self-Managed) is required for the initial GitHub configuration.
+
+:::
+
+1. Within Web Modeler, navigate to the process application you would like to connect to GitHub, and click **Connect GitHub**.
+
+2. Provide the following information in the GitHub Configuration modal:
+   - **App Owner:** Found in your GitHub App's settings page.
+   - **App ID:** Found in your GitHub App's settings page.
+   - **Private Key:** Found in your GitHub App's settings page.
+   - **GitHub repository URL:** The URL of the repository you would like to sync with.
+   - **Branch name:** The branch name to use for merging and managing changes.
+
+Click **Save Configuration**.
+:::note
+When synchronizing for the first time with a remote repository that already contains commits, ensure Web Modeler has assigned the correct main process.
```

Comment:
💭 maybe we could specify even more what `Web Modeler has assigned the correct main process` means? @JacobiCamunda any thoughts on how to improve it even more? I remember you verbalized this case in some past discussion.

Reply:
My verbalization was incorporated in the "File name conventions" part at the end of the document. I think we can keep it this way 👍

Reply:
But I think we should mention the initial sync and how it works a bit differently + it is done always when branch switching. What do you think, @marcellobarile?